index.html

```diff
@@ -1,92 +1,170 @@
+      cardiology, cardiovascular surgery and heart, lung, cardiopulmonary, liver, kidney, and bone marrow
+      transplants, among other areas.
+      Favaloro focused his career there, surrounded by a selected group of professionals. One of his more
+      famous
+      patients was boxing promoter and Luna Park arena owner Tito Lectoure, on whom Dr. Favaloro operated in
+      1990.
+      It should be
+      mentioned, however, that despite the immense costs of bypass surgery, Dr Favaloro operated daily on
+      indigent
+      patients, something that he felt was both a necessity and his obligation.</li>
+    <li class="h3-fix">Recognitions & distinctions</li>
+    <li><strong>1979 - John Scott Prize granted by Philadelphia</strong></li>
+    <li><strong>1980 - The creation of the Chair of Cardiovascular Surgery "Dr René G. Favaloro" (Tel Aviv
+        University, Israel)</strong></li>
+    <li><strong>1982 - The distinction of the Fundación Conchita Rábago de Giménez Díaz (Madrid,
+        Spain)</strong>
+    </li>
+    <li><strong>1986 - The Teacher Prize of Argentinian Medicine </strong></li>
+    <li><strong>1987 - The Distinguished Alumnus Award of the Cleveland Clinic Foundation </strong></li>
+    <li><strong>1987 - The Gairdner Foundation International Award (Toronto, Canada)</strong></li>
+    <li><strong>1989 - René Leriche Prize ,, granted by the International Surgery Society </strong></li>
+    <li><strong>1992 - The Gifted Teacher Award, granted by the American College of Cardiology </strong></li>
+    <li><strong>1993 - The Golden Plate Award of the American Academy of Achievement </strong></li>
+    <li><strong>1993 - The Diamond Konex Award as the most important scientist in the last decade in his
+        country
+        (Argentina)</strong></li>
+    <li><strong>1999 - The Prince Mahidol Award, granted by His Majesty the King of Thailand (Bangkok,
+        Thailand)
+      </strong>
+  </ul>
+  <br><br>
+  <i>"The progress of medicine and bioengineering can be considered true achievements for humanity when all
+    people
+    have access to its benefits and cease to be a privilege for minorities." (Buenos Aires, Congress of
+    Bioengineering 1999)</i>
+</div>
+<h5>If you have time, you should read more about this incredible human being on his <a id="tribute-link"
+    href="https://www.fundacionfavaloro.org/welcome-to-the-favaloro-foundation/" target="_blank"
+    rel="noopener noreferrer">Favaloro
+    Foundation</a>.</h5>
 
-      <div id="img-div">
-        <img id="image" src="https://lavozderosario.com/wp-content/uploads/2020/07/Dr._Ren%C3%A9_Favaloro-1.jpg" alt="Dr. Rene Favaloro">
-        <figcaption id="img-caption">Dr. Rene Favaloro was a pioneer in heart bypass surgery. Each day, thousands of people with heart disease have René Favaloro to thank for each heartbeat. </figcaption>
-      </div>
 
-      <div id="tribute-info">
```

Comment:
clean repeated <div>

Reply:
- [ ] Kindly use comments politely, as suggestions instead of orders, and make sure to ask the `why` question. Why are you requesting these changes?